AdMobAdapter/src/main/java/com/chartboost/mediation/admobadapter/AdMobAdapter.kt

```diff
@@ -381,11 +381,20 @@ class AdMobAdapter : PartnerAdapter {
                 val adview = AdView(context)
                 val partnerAd = PartnerAd(
                     ad = adview,
-                    details = emptyMap(),
+                    details = mutableMapOf(),
```

Comment:
Nit: To avoid making this mutable we could do this after the fact:

```kt
val details = if (request.format.key == "adaptive_banner") {
    mapOf(
        "banner_width_dips" to "${adSize.width}",
        "banner_height_dips" to "${adSize.height}"
    )
} else {
    emptyMap()
}

val partnerAd = PartnerAd(
    ad = adview,
    details = details,
    request = request,
)
```

Reply:
I like that so much more than what I wrote!